enable-vcp-image/enable-vcp-scripts/daemonset_pod.sh

```diff
@@ -67,42 +61,58 @@ PHASE=$DAEMONSET_SCRIPT_PHASE4
 update_VcpConigStatus "$POD_NAME" "$PHASE" "$DAEMONSET_PHASE_RUNNING" ""
 
 # Creating back up directory for manifest files and kubelet service configuration file.
-echo "[INFO] Creating directory: '/host/tmp/$POD_NAME' for back up of manifest files and kubelet service configuration file"
-mkdir /host/tmp/$POD_NAME
-if [ $? -eq 0 ]; then
-    echo "[INFO] Successfully created back up directory:" /host/tmp/$POD_NAME " on $NODE_NAME node"
-else
-    ERROR_MSG="Failed to create directory: '/host/tmp/${POD_NAME}' for back up of manifest files and kubelet service configuration file"
-    update_VcpConigStatus "$POD_NAME" "$PHASE" "$DAEMONSET_PHASE_FAILED" "$ERROR_MSG"
-    exit $ERROR_FAIL_TO_CREATE_BACKUP_DIRECTORY
+ls /host/tmp/$POD_NAME &> /dev/null
+if [ $? -ne 0 ]; then
+    echo "[INFO] Creating directory: '/host/tmp/$POD_NAME' for back up of manifest files and kubelet service configuration file"
+    mkdir /host/tmp/$POD_NAME
+    if [ $? -eq 0 ]; then
+        echo "[INFO] Successfully created back up directory:" /host/tmp/$POD_NAME " on $NODE_NAME node"
+    else
+        ERROR_MSG="Failed to create directory: '/host/tmp/${POD_NAME}' for back up of manifest files and kubelet service configuration file"
+        update_VcpConigStatus "$POD_NAME" "$PHASE" "$DAEMONSET_PHASE_FAILED" "$ERROR_MSG"
+        exit $ERROR_FAIL_TO_CREATE_BACKUP_DIRECTORY
+    fi
 fi
 
 # Verify that the directory for the vSphere Cloud Provider configuration file is accessible.
 ls /host/$k8s_secret_vcp_configuration_file_location &> /dev/null
 if [ $? -eq 0 ]; then
     echo "[INFO] Verified that the directory for the vSphere Cloud Provider configuration file is accessible. Path: ("/host/$k8s_secret_vcp_configuration_file_location ")"
 else
-    ERROR_MSG="Directory (/host/${k8s_secret_vcp_configuration_file_location}) for vSphere Cloud Provider Configuration file is not present"
-    update_VcpConigStatus "$POD_NAME" "$PHASE" "$DAEMONSET_PHASE_FAILED" "$ERROR_MSG"
-    exit $ERROR_VSPHERE_CONF_DIRECTORY_NOT_PRESENT
+    mkdir /host/$k8s_secret_vcp_configuration_file_location
```

Comment:
can you please use "mkdir -p".
Here and everywhere.

Reply:
fixed.